### PR TITLE
cloud: add the contract tab to billings

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-billing.md [hunk 1]
```diff
@@ -141,3 +141,14 @@ To edit the billing profile information, perform the following steps:
 2. Click **Billing**.
 3. Click the **Payment Method** tab.
 4. Edit the billing profile information, and then click **Save**.
+
+## Contract
+
+If you have agreed with our sales on a subscription and received an email to review and accept the agreement online, you can do the following:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:147]:
```suggestion
If you have agreed with our sales on a contract and received an email to review and accept the contract online, you can do the following:
```

File: tidb-cloud/tidb-cloud-billing.md [hunk 2]
```diff
@@ -141,3 +141,14 @@ To edit the billing profile information, perform the following steps:
 2. Click **Billing**.
 3. Click the **Payment Method** tab.
 4. Edit the billing profile information, and then click **Save**.
+
+## Contract
+
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:146]:
```suggestion
TiDB Cloud provides you with the contract management feature to make contract management easier and more secure. 
```

File: tidb-cloud/tidb-cloud-billing.md [hunk 3]
```diff
@@ -141,3 +141,16 @@ To edit the billing profile information, perform the following steps:
...
+TiDB Clould provides the online contracts feature to customize TiDB Cloud subscriptions and meet compliance requirements.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:147]:
```suggestion
TiDB Clould provides you with the online contracts feature to customize TiDB Cloud subscriptions and meet compliance requirements.
```

**Review by langyuezhang — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-billing.md [hunk 4]
```diff
@@ -141,3 +141,16 @@ To edit the billing profile information, perform the following steps:
 2. Click **Billing**.
 3. Click the **Payment Method** tab.
 4. Edit the billing profile information, and then click **Save**.
+
+## Contract
+
+TiDB Clould provides you with the online contracts feature to customize TiDB Cloud subscriptions and meet compliance requirements.
```
[the remark this answers is not in the document]
Comment (qiancai) [reply tidb-cloud/tidb-cloud-billing.md:147]: According to @langyuezhang, this feature acts as an additional customize subscription in addition to the online pricing and CLA, so could we update "manage your customized TiDB Cloud subscriptions"? @langyuezhang PTAL

File: tidb-cloud/tidb-cloud-billing.md [hunk 5]
```diff
@@ -141,3 +141,16 @@ To edit the billing profile information, perform the following steps:
 2. Click **Billing**.
 3. Click the **Payment Method** tab.
 4. Edit the billing profile information, and then click **Save**.
+
+## Contract
+
```
[the remark this answers is not in the document]
Comment (langyuezhang) [reply tidb-cloud/tidb-cloud-billing.md:146]: owner and billing admin can see it

File: tidb-cloud/tidb-cloud-billing.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (langyuezhang) [reply tidb-cloud/tidb-cloud-billing.md:147]: 可以的

File: tidb-cloud/tidb-cloud-billing.md [hunk 6]
```diff
@@ -141,3 +141,16 @@ To edit the billing profile information, perform the following steps:
...
+In the TiDB Cloud console, you can manage your TiDB Cloud subscriptions to meet compliance requirements.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:147]:
```suggestion
In the TiDB Cloud console, you can manage your customized TiDB Cloud subscriptions to meet compliance requirements.
```

**Comment by qiancai** [comment]
@ran-huang Thanks for the review. All your comments are implemented. PTAL again. Thanks.

**Review by ran-huang — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by langyuezhang — APPROVED** [review]
LGTM

**Pull request merged**